### PR TITLE
Feature host relay

### PR DESCRIPTION
**Pull request by abhishek9686**
-> Added APIs to create/delete host relay and respective functions to support relay operations on host
-> added relay fields to host model
-> relay information in host update for proxy
-> fixed node model conversion during node update

 **YET TO REMOVE OLD CODE RELATED TO NODE RELAYS** ( will be a separate PR)

### CODE REVIEW

File: models/host.go [hunk 1]
```diff
@@ -34,6 +34,10 @@ type Host struct {
 	TrafficKeyPublic []byte           `json:"traffickeypublic" yaml:"trafficekeypublic"`
 	InternetGateway  net.UDPAddr      `json:"internetgateway" yaml:"internetgateway"`
 	Nodes            []string         `json:"nodes" yaml:"nodes"`
+	IsRelayed        bool             `json:"isrelayed" bson:"isrelayed" yaml:"isrelayed"`
```
Comment (mattkasun) [inline models/host.go:37]: bson tags not required

File: models/structs.go [hunk 2]
```diff
@@ -172,6 +172,12 @@ type RelayRequest struct {
 	RelayAddrs []string `json:"relayaddrs" bson:"relayaddrs"`
 }
 
+// HostRelayRequest - struct for host relay creation
+type HostRelayRequest struct {
+	HostID       string   `json:"host_id" bson:"host_id"`
```
Comment (mattkasun) [inline models/structs.go:177]: bson tags not required

**Review by mattkasun — CHANGES_REQUESTED** [review]
run go mod tidy

**Review by 0xdcarns — APPROVED** [review]

**Comment by abhishek9686** [comment]
> run go mod tidy

updated go mod and sum

**Review by 0xdcarns — APPROVED** [review]

**Review by 0xdcarns — APPROVED** [review]

**Pull request merged**